wlroots/ffi_build.py

```diff
@@ -544,6 +544,25 @@
 };
 """
 
+# types/wlr_screencopy_v1.h
+CDEF += """
+struct wlr_screencopy_manager_v1 {
+    struct wl_global *global;
+    struct wl_list frames; // wlr_screencopy_frame_v1::link
+
+    struct wl_listener display_destroy;
+
+    struct {
+        struct wl_signal destroy;
+    } events;
+
+    void *data;
```

Comment:
If you add `...;` here, it will do some basic checking to be sure you have defined the struct in the same way as the bound type.